test/index.js

```diff
@@ -114,6 +114,18 @@ describe('frameguard', function () {
       assert.equal(directive, 'ALLOW-FROM')
       assert.equal(url, 'http://example.com')
     })
+
+    it('works with ALLOW-FROM with Array objects', function (done) {
```

Comment:
This tests that it chooses the first domain in the list. Could we also add another test (in a new `it` block) that it adds the right domain if `req.hostname` changes? It probably requires setting the `Host` header in the tests.